src/OpenTelemetry.Instrumentation.AspNet/OpenTelemetry.Instrumentation.AspNet.csproj

```diff
@@ -23,4 +23,8 @@
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule\OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
```

Comment:
this is really against OpenTelemetry principles, and should not be done.
Please do either of the below:
1. Just documenting this in readme and ask users to add views themselves.
2. Request to make OTel sdk give special treatment to this one also. (I am supportive of this idea).

Reply:
Yes, I figured that was the case, I just wanted to make the tests pass earlier, I completely agree it's not a good idea.

I removed it and went with option 2 for now, I opened at issue at https://github.com/open-telemetry/opentelemetry-dotnet/pull/5063 for now.

I have removed the dependency again but leaving the tests to fail until we decide whether to wait on the upstream PR.